### PR TITLE
Fix warning about IMG alt

### PR DESCRIPTION
**Pull request by Abyss777**
There was warning in console
````
[W] For WAI-ARIA compliance, IMG elements SHOULD have an alt attribute.
````

### CODE REVIEW

**Comment by tananaev** [comment]
I think it's better to rename it to "Logo" or something. Also, it should probably be in the strings file.

**Comment by Abyss777** [comment]
Done.

File: web/l10n/en.json [hunk 1]
```diff
@@ -58,6 +58,7 @@
     "loginFailed": "Incorrect email address or password",
     "loginCreated": "New user has been registered",
     "loginLogout": "Logout",
+    "loginLogo":"Logo",
```
Comment (tananaev) [inline web/l10n/en.json]: Space is missing :-)

**Comment by Abyss777** [comment]
Sorry, why I'm so inattentive sometime...

**Comment by tananaev** [comment]
Thanks for pull request, merged.

**Pull request merged**